src/TorchSharp/Utils/TensorAccessor.cs

```diff
@@ -47,6 +48,16 @@ public T[] ToArray()
             if (_tensor.ndim < 2)
                 return (T[])ToNDArray();
 
+            if (_tensor.is_contiguous()) {
```

Comment:
Is there a situation where the tensor is a view, still seen as 'contiguous,' but the offset of the first element of the tensor is not 0?

Reply:
Oh, i don't know, i will investigate that and add in Test Unit